tools/testing/gatekeeper/required-tests.yaml

```diff
@@ -0,0 +1,42 @@
+required_tests:
+  # Always required tests
+  - "Commit Message Check / Commit Message Check"
+required_regexps:
+  # Always required regexps
+
+paths:
+  # Mapping of path (python) regexps to set-of-tests (sort by order of importance)
+  # CI
+  - "^ci/openshift-ci/": []
+  - "^\\.github/workflows/": []
+  # TODO: Expand filters
+  # Documentation
+  #- "\\.rst$": ["build"]
+  #- "\\.md$": ["build"]
+  # Sources
+  #- "^src/": ["static", "build", "test"]
+
+mapping:
+  # Mapping of set-of-tests to required test names and/or test name regexps
+  # TODO: Modify this according to actual required tests
+  test:
+    # Checks the basic functional tests work
+    regexps: "Kata Containers CI / .*run-basic-amd64-tests.*|Kata Containers CI / .*run-metrics-tests.*"
+    names:
```

Comment:
How about using `tests` here to match `required_tests` (similar to the pair `required_regexps` and `regexps`)

Reply:
I could do but given the file is called `required-tests` I assumed this prefix by default (but maybe I'm just too lazy to type :-) )

Reply:
Just to explicitly state, I'm not changing this for the next version unless you (or someone) asks me to do so.